src/app/ui/palette_view.cpp

```diff
@@ -972,7 +972,8 @@ int PaletteView::findExactIndex(const app::Color& color) const
   switch (color.getType()) {
 
     case Color::MaskType:
-      return (current_editor ? current_editor->sprite()->transparentColor(): -1);
+      return (currentPalette()->findMaskColor() >= 0) ?
+        currentPalette()->findMaskColor() : -1;
```

Comment:
Minor change to avoid double findMaskColor (anyway given the regression I'm not 100% sure if this should be the final solution):
```suggestion
    case Color::MaskType: {
      int i = currentPalette()->findMaskColor();
      return (i >= 0 ? i: -1);
    }
```

Reply:
Yeah, I. think RGB and Grayscale should apply this. I'll fix soon.